packages/plugin-knowledgebase-api/src/models/Configs.ts

```
@@ -0,0 +1,153 @@
+      return {
+        accountId,
+        accessKeyId,
+        region: 'auto',
+        secretAccessKey,
+        bucket,
+        useCdn,
+        isPublic,
+        apiToken
+      };
+    }
+  }
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)